frontend/src/views/Pages/AccountOverview/GddTransactionList.vue

```diff
@@ -113,7 +122,11 @@
 
           <!-- Collaps Start -->
 
-          <b-collapse v-if="type != 'decay' && decay" class="pb-4" :id="'decay-' + transactionId">
+          <b-collapse
+            v-if="(type != 'decay' && decay) || firstTransaction"
```

Comment:
```suggestion
            v-if="(type !== 'decay' && decay) || firstTransaction"
```
It is two times the same code. Can we improve this?